python/paddle/fluid/transpiler/inference_transpiler.py

```diff
@@ -50,6 +51,7 @@ def transpile(self, program, place, scope=None):
             place (Place): inference place
             scope (Scope|None): inference Scope
         '''
+        sys.stderr.write('InferenceTranspiler is deprecated.\n')
```

Comment:
There should tell the user the new method that can replace the function of InferenceTranspiler.